metadb/scrapers/spotgen.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import spotipy
+
+
+sp = spotipy.Spotify()
+sp.trace = False
+
+def get_artist(name):
+	results = sp.search(q='artist:' + name, type='artist')
+	items = results['artists']['items']
+	if len(items) > 0:
+		return items[0]
+	else:
+		return None
+
+
+
+def scrape(mbid, artist, title, metadata=None):
+	data = None
+	spot_artist = get_artist(artist)
+	print('====', spot_artist['name'], '====')
+	if len(spot_artist['genres']) > 0:
+		print('Genres: ', ','.join(spot_artist['genres']))
+		data = spot_artist['genres']
```

Comment:
This is a good start, getting the genre of the artist, however I think we will find that not every artist performs in the same genre over all of their recordings. It would be good to get some more detailed information.
You're right that there isn't genre level information per track, but there is genre information for each release. I think you could try two options:

 * search for the `artist` and `title` together. If you find a match, find all of the releases that each of these recordings appear on, and return the genre of those releases.
 * Expand this scraper file to include a parameter with a release name. You will also need to update [the lookup script](https://github.com/MTG/metadb/blob/master/lookup.py#L20) to allow people to pass in a file or parameters with the release name.